src/types.ts

```diff
@@ -53,6 +53,8 @@ export type NoteMessage = {
   duration?: number;
   grace: boolean;
   graceSlash: boolean;
+  arpeggiate: boolean;
+  arpeggiateDirection?: string;
```

Comment:
This is fine for now, but we should start thinking about breaking up note messages into different subtypes. Perhaps like choices in the content of the [`<note>`](https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/note/) element.

Reply:
agree